Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -79,7 +79,7 @@ private Stream Stream
         {
             _uri = uri;
             _useTls = useTlsEncryption;
-            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(true);
+            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(false);
```

Comment:
Errrrr - if I'm totally honest with you I have no idea why that is set to `true` - I would have thought we'd have put it to `false`. There's no reason why we'd need to stay in the same SyncContext - I don't even know why we'd have done that in the first place. 

Does it crop up anywhere else in the codebase?